### DISCUSSION TITLE
How to get a minimal installation of `git`?

### DISCUSSION THREAD

**Discussion by just1a-person** [git-for-windows/git#3203] [Q&A] [answered]
Is there any command line arguments that I can pass to the git installer to get a bare installation of git for interactive usage (without git bash/git gui and other stuff)?

**Comment by landstander668** [comment, marked as answer]
As far as I know, the only way to do this is by installing the MinGit zipfile. I believe it's intended primarily for containerized environments, however, rather than interactive usage.

**Reply by dscho** [reply]
That's correct. If by "bare installation" you mean "a version of Git for Windows that does not provide any interactive functionality", MinGit is for you.

**Reply by just1a-person** [reply]
I mean "bare installation for interactive usage" (updated the description)

**Reply by dscho** [reply]
Then I don't understand. You need `git-bash.exe` for interactive usage. And Git GUI. Otherwise it's not Git for Windows.
